.travis.yml

```diff
@@ -31,11 +31,14 @@ script:
       (
         set -x;
         cargo fmt -- --write-mode=diff &&
-        cargo build &&
-        cargo test  --verbose
+        cargo rustc --verbose -- -Zno-trans &&
```

Comment:
I guess we want to build the lib in debug to check it compiles and then build and run the tests in release to check they compile too.